lib/src/models/gallery_photo_aggregator.dart

```diff
@@ -0,0 +1,18 @@
+
+import 'package:flutter_aira/src/models/gallery_photo.dart';
+
+class GalleryPhotoAggregator {
```

Comment:
Did you borrow this pattern from somewhere? I haven't seen it before. 

Rather than having the client pass in an aggregator, I'd prefer something like this:

```dart
Future<PhotosPage> getPhotosPage(int page) async {
```

```dart
class PhotosPage {
  PhotosPage({
    required this.page,
    required this.hasMore,
    required this.photos,
  });

  final int page;
  final bool hasMore;
  final List<Photo> photos;
}
```